### PR TITLE
RavenDB-15561 MinimumRevisionsToKeep - Refactor info text

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts [hunk 1]
```diff
@@ -41,16 +42,49 @@ class revisionsConfigurationEntry {
         this.initValidation();
     }
     private initObservables() {
+        this.deleteDescription = ko.pureComputed(() => {
+            const purgeOffText = `<li>A revision will be created anytime a document is modified or deleted.</li>
+                                  <li>Revisions of a deleted document can be accessed in the Revisions Bin view.</li>`;
+
+            const purgeOnText = `<li>A revision will be created anytime a docment is modified.</li>
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: typo

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts [hunk 2]
```diff
@@ -41,16 +42,49 @@ class revisionsConfigurationEntry {
...
+            const numberOnlyText = (numberToKeep: number) => 
+                `<li>Only the latest <strong>${numberToKeep}</strong> revisions will be kept.</li>
+                 <li>Older revisions will be removed on next revision creation.</li>`;
+            
+            const ageOnlyText = (retentionText: string) => 
+                `<li>Revisions that exceed <strong>${retentionText}</strong> will be removed on next revision creation.</li>`;
+            
+            const numberAndAgeText = (numberToKeep: number, retentionText: string) => 
+                `<li>At least <strong>${numberToKeep}</strong> of the latest revisions will be kept.</li>
+                 <li>Older revisions will be removed if they exceed <strong>${retentionText}</strong> on next revision createion.</li>`;
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: typo

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts [hunk 3]
```diff
@@ -41,16 +42,49 @@ class revisionsConfigurationEntry {
...
-                `Revisions are going to be removed on next revision creation or document deletion once they exceed retention time of <strong>${retentionTimeHumane}</strong>` : "";
+            const limitByNumber = this.limitRevisions() && this.minimumRevisionsToKeep.isValid();
+            const limitByAge = this.limitRevisionsByAge() && this.minimumRevisionAgeToKeep.isValid();
+            
+            let description;
             
-            const countPart = this.limitRevisions() && this.minimumRevisionsToKeep.isValid() ? 
-                `At least <strong>${this.minimumRevisionsToKeep()}</strong> revisions are going to be kept` : "";
+            if (limitByNumber && !limitByAge) {
+                description = numberOnlyText(this.minimumRevisionsToKeep());
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: inline those function here - no need to have those functions.

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: done

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: done

File: src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/documents/revisionsConfigurationEntry.ts]: done

**Review by ml054 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**